src/adapters/cache.ts

```diff
@@ -88,33 +99,40 @@ class Sink {
   }
 }
 
-export class FileHandle {
-  constructor (path, cache) {
+export class FileHandle implements FileSystemFileHandleAdapter {
+  readonly kind = 'file'
+  private _cache: Cache
+  private path: string
```

Comment:
note to @use-strict typescript users should stop using `private` and switch to using `#` instead